### PR TITLE
Adds Winter 2022 GM Page

### PR DESCRIPTION
**Pull request by advaithg**
adds GM page at `/gm/w22` (with a redirect from `/wintergm`)
changes banner on homepage

### CODE REVIEW

**Review by khanhzoball — COMMENTED** [review]
Looks good!

**Review by christinatong01 — APPROVED** [review]
Looks good to me too! The Google Slides icon works for now. Thanks!

File: _redirects [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /fallgm         /gm/f21
 /town-hall      /town-hall/f21
+/wintergm       /gm/w22
```
Comment (evanzhong) [inline _redirects:3]: Add newline here

File: pages/gm/w22.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+import React from 'react';
+import Countdown from 'react-countdown';
+
+import Banner from '../../components/Banner';
+import Layout from '../../components/Layout';
+
+const GM_START_TIME = new Date('2022-01-06T18:30-07:00');
+const RSVP_LINK = 'http://links.uclaacm.com/wintergm-rsvp-22';
+const SLIDES_LINK = 'https://docs.google.com/presentation/d/1XGyJdmuR78C9liEPlwqMq2PpNSaXS-PtlK15fvzQLRE/edit?usp=sharing';
+// const ZOOM_LINK = '#';
```
Comment (evanzhong) [inline pages/gm/w22.js:12]: Assuming this will be added in followupPR

File: pages/gm/w22.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
+import { NextSeo } from 'next-seo';
+import Link from 'next/link';
+import React from 'react';
+import Countdown from 'react-countdown';
+
+import Banner from '../../components/Banner';
+import Layout from '../../components/Layout';
+
+const GM_START_TIME = new Date('2022-01-06T18:30-07:00');
```
Comment (evanzhong) [inline pages/gm/w22.js:9]: Can you recheck this start-time? It isn't rendering correctly on the demo site

**Review by evanzhong — CHANGES_REQUESTED** [review]
Light review, didn't check most of the code.

A couple things:
- Can the relevant information button section be re-styled (Discord has its own background color and you can use a simple grey for the google slides button)
    - image
- Double check the start-time, it's not rendering the correct countdown on my client
    - image

File: pages/gm/w22.js (hunk 2 above)
Comment (advaithg) [reply pages/gm/w22.js:12]: Yep!

**Comment by advaithg** [comment]
Is there a reason you want the button backgrounds to be changed?
Additionally, should I also change the zoom background to light grey? That won't work later since it'll be replaced by the logo, which is white.

**Comment by evanzhong** [comment]
Button background changes are purely stylistic. Technically since we are using Discord and Google Slide's logos we should adopt their style guides (https://discord.com/branding)

There should be some styles you can reference in https://github.com/uclaacm/website/pull/66

**Review by evanzhong — APPROVED** [review]
lgtm! Great work!

**Pull request merged**